Lib/test/test_peg_parser.py

```diff
@@ -1,803 +0,0 @@
-import ast
```

Comment:
why is this file deleted?

Reply:
This file is comparing the ast of the old parser to the last of the new parser, so it serves no purpose anymore. We have been adding all negative cases (testing extensions) into other files so we won't lose anything